server/controller/user-controller.js

```diff
@@ -26,27 +27,80 @@ var tbl_user = models.tbl_user;
  * @apiSuccess {string} message Message corresponding to request.
 */
 exports.signup = function (request, response) {
-
-	var user = request.body;
-	if(common.required(user.first_name) && common.required(user.last_name) && common.required(user.password) && common.required(user.email) && common.required(user.gender) && common.required(user.age) && common.required(user.phone)) {
-	    //if email exist, send response back
+    var user = request.body;
+    if(common.required(user.first_name) && common.required(user.last_name) && common.required(user.password) && common.required(user.email) && common.required(user.gender) && common.required(user.age) && common.required(user.phone)) {
+        //if email exist, send response back
         tbl_user.findAll({ where: { email: user.email } }).then(function (res) {
             if(res.length > 0) {
                 common.sendResponseBack(response, 'FAIL', 'This email is already linked with another account. Please use different email.', null);
             } else {
-            	user.role = 'USER';
-            	user.isDeleted = 0;
-            	user.registerDate = new Date();
-            	user.password = md5(user.password);
-            	tbl_user.build(user).save().then(function(result) {
-			    	common.sendResponseBack(response, 'OK', 'User signup successfully!', null);
-			    }, (error) => {
-			    	common.sendResponseBack(response, 'FAIL', 'Some error occured while processing your request, Please try again later.', null);
+                user.role = 'USER';
+                user.isDeleted = 0;
+                user.registerDate = new Date();
+                user.password = md5(user.password);
+                tbl_user.build(user).save().then(function(result) {
+                    common.sendResponseBack(response, 'OK', 'User signup successfully!', result);
+                }, (error) => {
+                    common.sendResponseBack(response, 'FAIL', 'Some error occured while processing your request, Please try again later.', null);
                     logger.error( 'Error occured on '+new Date()+' with reason' + error);
-			    });
+                });
             }
         });
-	} else {
-        common.sendResponseBack(response, 'FAIL', 'Please fill all the fields first.', null);
     }
-}
\ No newline at end of file
+}
+
+/**
+ * @api {get} user/login Login API
+ * @apiName Login API
+ * @apiGroup User
+ *
+ * @apiParam {string} password Password
+ * @apiParam {string} email Email Address
+ *
+ * @apiSuccess {string} status Status of the request.
+ * @apiSuccess {string} message Message corresponding to request.
+*/
+exports.login = function (request, response) {
+    
+    var user = request.query;
+    tbl_user.findAll({ where: { email: user.email, password: user.password } }).then(function (res){
+         if(res.length > 0) {
+                common.sendResponseBack(response, 'PASS', 'YOUR EMAIL AND PASSWORD ARE CORRECT WAIT FEW SECONDS', null);
```

Comment:
Change the message and status it should be like this
common.sendResponseBack(response, 'OK', 'User logged in successfully!', null);